content/en/docs/reference/instrumentation/metrics.md

```diff
@@ -2533,7 +2533,7 @@ Alpha metrics do not have any API guarantees. These metrics must be used at your
 <tr class="metric"><td class="metric_name">pod_security_errors_total</td>
 <td class="metric_stability_level" data-stability="alpha">ALPHA</td>
 <td class="metric_type" data-type="counter">Counter</td>
-<td class="metric_description">Number of errors preventing normal evaluation. Non-fatal errors may result in the latest restricted profile being used for evaluation.</td>
+<td class="metric_description">Number of errors preventing normal evaluation. Non-fatal errors may result in the latest restricted profile being used for enforcement.</td>
```

Comment:
This page is automatically generated. That change actually needs to happen separately; the file to change is https://github.com/kubernetes/kubernetes/blob/6fa2df07d6ea46d1a8c5c731f3de66e9e912eaa8/test/instrumentation/documentation/documentation-list.yaml#L4672 (I think).